src/_pyinstaller_hooks_contrib/hooks/stdhooks/hook-pyqtgraph.py

```diff
@@ -0,0 +1,16 @@
+# ------------------------------------------------------------------
+# Copyright (c) 2021 PyInstaller Development Team.
+#
+# This file is distributed under the terms of the GNU General Public
+# License (version 2.0 or later).
+#
+# The full license is available in LICENSE.GPL.txt, distributed with
+# this software.
+#
+# SPDX-License-Identifier: GPL-2.0-or-later
+# ------------------------------------------------------------------
+
+from PyInstaller.utils.hooks import collect_data_files
+
+# Data files
+datas = collect_data_files("pyqtgraph", include_py_files=True)
```

Comment:
What's the reasoning behind `include_py_files=True`? It's a pretty unusual thing to do. The only use case I know for it is if Python scripts are used as templates for writing other Python files. If you get `ModuleNotFoundError`s without it then they should usually be hidden imports instead.

Reply:
OK, I didn't know that. I just thought this would be future-proof, because pyqtgraph is still a moving target. If you insist, I will change this to hidden imports.

Reply:
I probably should have at least put that in the comment instead of `# Data files`

Reply:
```python
hiddenimports = collect_submodules("pyqtgraph")
```
Would be to future-proof against hidden imports but I wouldn't do that unless `pyqtgraph` are known to do nasty stuff with their imports.

Reply:
Apparently, it only affects a few Python files which all have "Template" in their name. I updated the hook and elaborated there what I did.